src/index.ts

```diff
@@ -365,6 +367,11 @@ function getConfig(userConfig?: Partial<Config>): Config {
     config.captureLambdaPayloadMaxDepth = Number(getEnvValue(captureLambdaPayloadMaxDepthEnvVar, "10"));
   }
 
+  if (userConfig === undefined || userConfig.localTesting === undefined) {
+    const result = getEnvValue(localTestingEnvVar, "false").toLowerCase();
+    config.localTesting = result === "true";
```

Comment:
The extension also allows for [`DD_LOCAL_TEST=1`](https://github.com/DataDog/datadog-agent/blob/10907e0430601006ed896982a9aeacbff1ffdf8e/pkg/serverless/daemon/routes.go#L41)

Reply:
Fixed – but this shouldn't ever have been done and should be removed.